packages/auth/package.json

```diff
@@ -12,6 +12,7 @@
     "build": "pnpm build:deps && pnpm clean && tsc --build tsconfig.json && pnpm copy-files",
     "clean": "rm -fr dist/",
     "test": "jest --passWithNoTests --maxWorkers=50%",
+    "test:cov": "pnpm test -- --coverage --changedSince=main",
```

Comment:
Keep in mind that this will only test things that jest thinks changed. So it does not just run all the tests like `test` does. https://jestjs.io/docs/cli#--changedsince

In limited testing this seems to include. a lot (1 line change in auth grant service tested 10/14 tests)

I also considered adding a `test:sincemain` that runs all tests with `--changedSince=main`. Then adding `test:sincemain:cov` and `test:cov`.

Reply:
> I also considered adding a `test:sincemain` that runs all tests with `--changedSince=main`. Then adding `test:sincemain:cov` and `test:cov`.

I think that's a good idea.

Reply:
added